src/components/proofs/ProofList.tsx

```diff
@@ -1,45 +1,54 @@
 import { AccentText, BodyText } from 'components/Text'
-import { Suspense } from 'preact/compat'
-import { space } from 'classnames/tailwind'
+import { CategoriesTitles, categories } from 'models/Categories'
+import { Suspense, useState } from 'preact/compat'
+import {
+  classnames,
+  display,
+  flex,
+  flexDirection,
+  gap,
+  space,
+} from 'classnames/tailwind'
 import { useSnapshot } from 'valtio'
-import ERC721ProofSection from 'components/proofs/ERC721ProofSection'
-import EmailProof from 'components/proofs/EmailProof'
-import Network from 'models/Network'
-import ProofSection from 'components/ProofSection'
+import DesktopProofCategories from 'components/proofs/DesktopProofCategories'
+import MobileProofCategories from 'components/proofs/MobileProofCategories'
 import ProofStore from 'stores/ProofStore'
-import ReadyEmailProof from 'components/proofs/ReadyEmailProof'
 import Scrollbar from 'components/Scrollbar'
 import WalletStore from 'stores/WalletStore'
 
+const proofList = classnames(
+  flex('flex-1'),
+  flexDirection('flex-col'),
+  space('space-y-2')
+)
+const menuWrapper = classnames(display('flex'), gap('gap-x-4'))
+
 export function ProofListSuspended() {
   const { account } = useSnapshot(WalletStore)
   const { emailProofsCompleted, proofsCompleted } = useSnapshot(ProofStore)
+  const [category, setCategory] = useState<CategoriesTitles>('NFTs')
 
   return (
     <>
+      <MobileProofCategories
```

Comment:
I need to place `MobileProofCategories` and `DesktopProofCategories` in different places, because wrapping `Scrollbar` into `div` will break it, and placing the `MobileProofCategories` inside the `Scrollbar` will cause it to scroll